pulsar-broker/src/test/java/org/apache/pulsar/schema/Schemas.java

```diff
@@ -16,25 +16,31 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pulsar.schema.compatibility;
+package org.apache.pulsar.schema;
 
 import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 import org.apache.avro.reflect.AvroDefault;
 
 public class Schemas {
 
     @Data
     @NoArgsConstructor
     @AllArgsConstructor
+    @Getter
```

Comment:
The @Data annotation includes @Getter and @Setter

Reply:
I will fix it, thank for your review:)